res/css/style.css

```diff
@@ -98,8 +99,12 @@ body,
   left: 0;
 }
 
+.treeViewBodyInner1 {
+  width: 100%;
```

Comment:
can you please explain why you had to add these `width: 100%`? What do they fix? Please add also the explanation as comment in the code :) Thanks!

Reply:
Julie, I removed that line! Please see explanation further below.